java/code/src/com/suse/manager/webui/services/pillar/MinionGeneralPillarGenerator.java

```diff
@@ -63,6 +63,7 @@
 
         pillar.add("contact_method", minion.getContactMethod().getLabel());
         pillar.add("mgr_server", minion.getChannelHost());
+        pillar.add("hub_server", minion.getHubServerHost());
```

Comment:
I think in most cases hub_server is set with a not hub server name.
I think this is not what we should do.

Reply:
Currently we have the field `mgr_server` which points to the server or to the nearest proxy. I'd actually expect it to point always to the actual Uyuni server, as the name suggests, but that's not the case.
For configuring Prometheus targets I need information where the server runs. I've been thinking about the appropriate name for the new field and `hub_server` is the best which came to my mind.